app/questionnaire/router.py

```diff
@@ -35,7 +40,7 @@ def __init__(
         )
 
     @property
-    def enabled_section_ids(self):
+    def enabled_section_ids(self) -> List:
```

Comment:
Should be more specific for container types, also we should use the built-in `list` over `typing.List`.

Reply:
Changed to `list`, thanks!

Reply:
The type the container returns needs to be specified, so we know what sort of list we get.

Reply:
Added, thanks!